src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const succesHandler = (message) => {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.className = 'message';
+  messageDiv.textContent = message;
+  document.body.appendChild(messageDiv);
+};
+
+const errorHandler = (error) => {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.className = 'message error-message';
+  messageDiv.textContent = error.message;
+  document.body.appendChild(messageDiv);
+};
+
+promise1.then(succesHandler).catch(errorHandler);
+promise2.then(succesHandler).catch(errorHandler);
```

Comment:
The function `succesHandler` is called here, but it contains a typo. Ensure that the function name is corrected to `successHandler`.